diff --git a/packages/canard/schema-form/src/core/settle/utils/commit/markCommitDeliveries.ts b/packages/canard/schema-form/src/core/settle/utils/commit/markCommitDeliveries.ts
--- a/packages/canard/schema-form/src/core/settle/utils/commit/markCommitDeliveries.ts
+++ b/packages/canard/schema-form/src/core/settle/utils/commit/markCommitDeliveries.ts
@@ -113,1 +113,1 @@
-    if (additional && globalNodes.has(candidate)) continue;
+    if (additional && false && globalNodes.has(candidate)) continue;
